index.html

```diff
@@ -72,7 +68,7 @@ <h3 class="advantages-title">Мало туристов</h3>
           <p class="advantages-inner">Большинство едет в Гранд Каньон и толпится там</p>
         </li>
         <li class="advantages-item">
-          <img class="advantages-img" src="#" alt="Вид на мост Дьявола.">
+          <img class="advantages-img" src="images/advantages/bridge.jpg" widht="800" height="385" alt="Вид на мост Дьявола.">
```

Comment:
```suggestion
          <img class="advantages-img" src="images/advantages/bridge.jpg" width="800" height="385" alt="Вид на мост Дьявола.">
```
Ошибка в написании атрибута `width`.